server/src/logic.js

```diff
@@ -72,7 +72,10 @@ export const userHandler = {
         where: { groupId: groups.map(g => g.id) },
       }));
   },
-  schedules(user) {
+  schedules(user, args) {
+    if (args.id) {
+      return Schedule.findAll({ where: { id: args.id } });
```

Comment:
this should only ever return schedules that the user is part of because this is exposed from a user. If you have a valid reason to filter so that it returns a user's schedule by ID then you should add `id` to the where clause below on L78. Otherwise (preferred) just make your client use the top level query which will go through the logic at `scheduleHandler.query` on L199.